### PR TITLE
Reduce property model fields 

### PR DESCRIPTION
**Pull request by LBHmsaitz**
The fields of the property model have been reduced and it now contains the most used according to the results of the spike APIF-70 https://docs.google.com/spreadsheets/d/1QA1sOsNStGtBiwlnVuGoLkQXhyRA7zbojecuS2CONFU/edit#gid=0

### CODE REVIEW

**Comment by LBHmsaitz** [comment]
Didn't check tests! Looking into it now

**Comment by bjpirt** [comment]
I thought this was getting ready to roll out under a v2 API - are we proposing changing v1 endpoints in this way?

**Comment by cob16** [comment]
> I thought this was getting ready to roll out under a v2 API - are we proposing changing v1 endpoints in this way?

That would also be a safer option

File: property-api/V1/Domain/Property.cs [hunk 1]
```diff
@@ -6,251 +6,25 @@ namespace property_api.V1.Domain
...
-        public string LevelCode { get; set; }
-        public string MajorRef { get; set; }
-        public string ManScheme { get; set; }
+        public string PropertyRef { get; set; }
+        public string HierarchyCode { get; set; }
+        public string ParentRef { get; set; }
+        public string RepairCostCode { get; set; }
         public string PostCode { get; set; }
-        public string PostDesig { get; set; }
-        public string ShortAddress { get; set; }
```
Comment (alexdesi) [inline property-api/V1/Domain/Property.cs]: Income collection team might need `ShortAddress` and some other fields.
We should keep it. You can do futher cleaning up after the endpoint is actually used.

File: property-api/V1/Domain/Property.cs [hunk 2]
```diff
@@ -6,251 +6,25 @@ namespace property_api.V1.Domain
...
-        public string Telephone { get; set; }
```
Comment (alexdesi) [inline property-api/V1/Domain/Property.cs]: potentially useful

File: property-api/V1/Domain/Property.cs [hunk 3]
```diff
@@ -6,251 +6,25 @@ namespace property_api.V1.Domain
...
-        public string CookingFuel { get; set; }
-        public string CompAvail { get; set; }
-        public string CompDisplay { get; set; }
-        public short NoSingleBeds { get; set; }
-        public short NoDoubleBeds { get; set; }
-        public bool OnlineRepairs { get; set; }
-        public string VmPropref { get; set; }
-        public bool? VoidmanLive { get; set; }
-        public bool Repairable { get; set; }
-        public string Address1 { get; set; }
```
Comment (alexdesi) [inline property-api/V1/Domain/Property.cs]: I am pretty sure we need it

File: property-api/V1/Domain/Property.cs [hunk 4]
```diff
@@ -6,251 +6,25 @@ namespace property_api.V1.Domain
...
-        public decimal? Accomfund { get; set; }
-        public decimal? Candsfund { get; set; }
-        public int? PropertySid { get; set; }
-        public int? Keys { get; set; }
-        public string Company { get; set; }
-        public string LettArea { get; set; }
-        public bool? RtbApplication { get; set; }
-        public bool? NoMaint { get; set; }
-        public string Maintresp { get; set; }
-        public bool? Leasehold { get; set; }
```
Comment (alexdesi) [inline property-api/V1/Domain/Property.cs]: this might be useful to determine which address to use

**Comment by bjpirt** [comment]
I think we need to treat this as the first v2 API and make sure it fits into the domain model and meets the guidelines we've been discussing. Perhaps we could discuss after the refinement tomorrow?

**Comment by LBHmsaitz** [comment]
Changes related to this PR will be reviewed for the V2 API

**Pull request closed without merging**